Resources/Prototypes/Entities/Mobs/NPCs/dino.yml

```diff
@@ -0,0 +1,523 @@
+- type: entity
+  save: false
+  parent: [ BaseSimpleMob, MobCombat ]
+  id: BaseMobDinosaur
+  suffix: AI
+  description: Enigmatic behemoths from a bygone era, known for their imposing stature and formidable presence.
+  abstract: true
+  components:
+  - type: NpcFactionMember
+    factions:
+    - Xeno
+  - type: Tool
+    speed: 1.5
+    qualities:
+      - Prying
+  - type: Prying
+    pryPowered: false
+    useSound: /Audio/Items/jaws_pry.ogg
+  - type: HTN
+    rootTask:
+      task: XenoCompound #maybe someday... But for now it's enough.
+    blackboard:
+      NavSmash: !type:Bool
+        true
+  - type: Sprite
+    sprite: Mobs/Aliens/Dinos/anky.rsi
+    layers:
+      - map: [ "enum.DamageStateVisualLayers.Base" ]
+        state: anki
+  - type: SolutionContainerManager
+  - type: ReplacementAccent
+    accent: dino
+  - type: Bloodstream
+    bloodReagent: DinoBlood
+    bloodMaxVolume: 75 #you don't want the map to become pools of blood
+    bloodlossDamage:
+      types:
+        Bloodloss:
+          0.5
+    bloodlossHealDamage:
+      types:
+        Bloodloss:
+          -1
```

Comment:
Your larger / bigger healthpool mobs are going to die of bloodloss a lot easier if you have their blood level this low without changing anything else in bloodstream, especially when they have a higher health pool to take more damage.

Bloodloss damage scales with % blood missing, and it's not that hard to lose 75 blood as a mob that can't tend it's own wounds (ai or player).

having more than or equal to 300 on those isn't a problem (it's also more fun for janitors anyways)

Reply:
I wanted to make more blood, but I noticed this comment in "argocyte" ( https://github.com/space-wizards/space-station-14/blob/master/Resources/Prototypes/Entities/Mobs/NPCs/argocyte.yml ), even Leviathan has this amount of blood. If you really think it's worth adding, then I'll correct it to more relevant values.

Reply:
I mustve missed those when I changed bloodstream stuff but yes they'd have the same problem

They aren't used in game yet thankfully unless I missed something

Reply:
Done